org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -152,4 +152,13 @@
       }
     }
   }
+
+  .collapsible-title {
```

Comment:
Might be worth spending 10 minutes checking now, because usually this "will do later" improvements will never be done